internal/conf/config.go

```diff
@@ -9,6 +9,13 @@ import (
 	"github.com/spf13/viper"
 )
 
+const (
+	// ClusterTypeDefault defines the default value for cluster type
+	ClusterTypeDefault = "default"
+	// ClusterTypeOpenshift defines the type-name for OpenShift clusters
+	ClusterTypeOpenshift = "openshift"
```

Comment:
This is super-duper nitpicky, and I clearly didn't notice it last time, but shouldn't it be spelled `ClusterTypeOpenShift`?

Reply:
ACK. Will fix